src/ORM/Query.php

```diff
@@ -793,6 +930,10 @@ public function __clone()
      */
     public function count()
     {
+        return $this->getCount();
+    }
+
+    public function getCount() {
```

Comment:
I think `count()` is also good because here a setter would not make sense. It will also be in line with the sqlish functions you can create via cakephp3 orm: https://github.com/cakephp/cakephp/blob/master/src/ORM/Query.php#L43 <- that would not become `getMin()`

@lorenzo what's your take here?